src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -999,8 +999,8 @@ ElectPowerLoadCenter::ElectPowerLoadCenter(EnergyPlusData &state, int const obje
     SetupOutputVariable("Electric Load Center Requested Electricity Rate", OutputProcessor::Unit::W, totalPowerRequest_, "System", "Average", name_);
 
     if (DataGlobals::AnyEnergyManagementSystemInModel && storagePresent_) {
-        SetupEMSActuator("Electrical Storage", name_, "Power Draw Rate", "[W]", eMSOverridePelFromStorage_, eMSValuePelFromStorage_);
-        SetupEMSActuator("Electrical Storage", name_, "Power Charge Rate", "[W]", eMSOverridePelIntoStorage_, eMSValuePelIntoStorage_);
+        SetupEMSActuator("Electrical Storage", name_, "Draw Electricity Rate", "[W]", eMSOverridePelFromStorage_, eMSValuePelFromStorage_);
+        SetupEMSActuator("Electrical Storage", name_, "Charge Electricity Rate", "[W]", eMSOverridePelIntoStorage_, eMSValuePelIntoStorage_);
```

Comment:
I don't see doc changes for these two. Maybe these should just stay as-is. Power is generic enough.

Reply:
Again my vote would be to revert this change for today to avoid having to fix multiple other places.